Django_GAE/guestbook/model.py

```diff
@@ -0,0 +1,41 @@
+from google.appengine.ext import ndb
+from google.appengine.api import memcache
+import logging
+
+DEFAULT_GUESTBOOK_NAME = 'de_name'
+
+# We set a parent key on the 'Greetings' to ensure that they are all in the same
+# entity group. Queries across the single entity group will be consistent.
+# However, the write rate should be limited to ~1/second.
+
+class Author(ndb.Model):
+    """Sub model for representing an author."""
```

Comment:
fix tab indent for all file Django_GAE/guestbook/*.py
- replace 4 space with 1 tab